internal/users/manager.go

```diff
@@ -262,6 +280,10 @@ func (m *Manager) startUserCleanupRoutine(cacheDir string, opts *options) {
 			case <-m.doClear:
 				func() {
 					m.cache.ClearAndRebuild(cacheDir)
```

Comment:
private function that is called in 2 placces? (once in New if error is of type needsrebuild and once here)